### PR TITLE
Add Blockmonsters (BEP20)

### PR DESCRIPTION
**Pull request by blockmonsters**
Adding info for BEP20 token 'Blockmonsters'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by blockmonsters** [comment]
@Cryptocool1 the discord link does not work now that "gg" was changed to "com"

File: blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "name": "twitter",
+            "url": "https://twitter.com/Block_Monsters"
+        },
+        {
+            "name": "telegram",
+            "url": "https://t.me/BlockMonsters"
+        },
+        {
+            "name": "discord",
+            "url": "https://discord.com/dYyK2MGw"
```
Comment (blockmonsters) [inline blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json:22]: @Cryptocool1 com does not work

File: blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        },
+        {
+            "name": "reddit",
+            "url": "https://reddit.com/BlockMonsters_"
```
Comment (blockmonsters) [inline blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json:26]: this should be https://www.reddit.com/r/BlockMonsters_

@Cryptocool1

Comment (blockmonsters) [reply blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json:26]: the /r was removed

Comment (Cryptocool1) [reply blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json:26]: Снимок экрана 2021-11-03 в 23 01 20

Comment (blockmonsters) [reply blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json]: there needs to be the underscore at the end with the slash

https://www.reddit.com/r/BlockMonsters_/

**Comment by blockmonsters** [comment]
@Cryptocool1 let me know if there is anything left to do on the PR. I will send the BNB within the next few hours.

File: blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json (hunk 1 above)
Comment (blockmonsters) [reply blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json]: ok what you supplied with /invite works. thanks

File: blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json (hunk 2 above)
Comment (blockmonsters) [reply blockchains/smartchain/assets/0x287Db351d5230716246CfB46AF8153025eDa6A0a/info.json]: i went ahead and updated it.

**Comment by Cryptocool1** [comment]
Hello, 
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by blockmonsters** [comment]
@Cryptocool1 

• What is the purpose of this token? How will it benefit Trust Wallet users?
**To serve as an investment vehicle for the Blockmonsters game / nft marketplace. Trust wallet users will be able to find the token easily and see their balance within the app.**
• How is the token being distributed? Are you conducting an airdrop?
**Via pancake swap**
• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
**We did a presale initially**
• Is the token listed on a centralized/decentralized exchange?
**Not currently**
• Is a Smart Contract audit completed? Where can we find this information?
**Yes it has been completed. See link here:** https://solidity.finance/audits/BlockMonsters/

**Comment by Cryptocool1** [comment]
Submission looks good, awaiting for fee payment

**Comment by Cryptocool1** [comment]
TWT-BEP2 is supported (Binance Chain), TWT-BEP20 version on Smart Chain is not.

**Pull request merged**